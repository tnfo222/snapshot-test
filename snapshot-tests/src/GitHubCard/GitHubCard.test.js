import gitHubCard from './GitHubCard';
import renderer from 'react-test-renderer';

test('renders a snapshot', () => {
    const tree = renderer.create(<gitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
});