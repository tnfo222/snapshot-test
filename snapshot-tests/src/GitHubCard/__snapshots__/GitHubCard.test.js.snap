// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `<gitHubCard />`;
